pages/linux/firewall-cmd.md

```diff
@@ -10,6 +10,10 @@
 
 `firewall-cmd --list-all`
 
+- Permanently move the interface into the trusted zone, effectively allowing all communication:
```

Comment:
Ooh errr. This example sounds dangerous - are you sure it's a good fit for the tldr-page?

Reply:
What does it do exactly?

Reply:
It looks like it disables a the firewall completely for a given network interface.

Reply:
I agree with you, if that's the case. In any case, you should request changes here so the PR can be put on hold.

Reply:
Good point, I can change to a different zone. Perhaps the default zone `block` is a good candidate?